eng/targets/Helix.props

```diff
@@ -20,6 +20,12 @@
     <NodeVersion>10.15.3</NodeVersion>
   </PropertyGroup>
 
+  <!-- Items with the type "HelixProperties" will become arbitrary JSON associated with the job -->
+  <ItemGroup>
+    <HelixProperties Condition="'$(BUILD_BUILDNUMBER)' != ''" Include="buildNumber" Value="$(BUILD_BUILDNUMBER)" />
+    <HelixProperties Condition="'$(BUILD_DEFINITIONNAME)' != ''" Include="buildDefinition" Value="$(BUILD_DEFINITIONNAME)" />
+  </ItemGroup>
```

Comment:
Should we move the quarantined/unquarantined/target framework here instead of in the test name as well?

Reply:
Yep, just had the same thought :). TFM and Project name need to stay where they are since these are Job-level metadata (and Project/TFM are WorkItem metadata)